docs/changelog.rst

```diff
@@ -3,7 +3,13 @@ Changelog
 =========
 
 * :feature:`2034` Update WebUI's design.
-* :feature:`439` Limit the number of pending transfers per channel.
+
+* :release:`0.5.1 <2018-08-17>`
+
+* :feature:`1898`: Improve the event formatting in the REST API
```

Comment:
Mh, should there be a colon after the issue number? We don't have it in the older releases

Reply:
We have it in the releases but not the issues. Let me correct.